docker-compose.kafka-queue.yml

```diff
@@ -0,0 +1,110 @@
+version: "3.2"
+services:
+    gateway:
+        volumes:
+            - "/var/run/docker.sock:/var/run/docker.sock"
+        ports:
+            - 8080:8080
+        image: functions/gateway:latest-dev
+        networks:
+            - functions
+        environment:
+            dnsrr: "true"  # Temporarily use dnsrr in place of VIP while issue persists on PWD
+            faas_kafka_brokers: kafka1:19092,kafka2:19092
+            faas_queue_topics: faas-request # ,other-topics
+        depends_on:
```

Comment:
Please remove the "depends_on" setting since we just found out it's deprecated anyway.